recipes-core/packagegroups/packagegroup-ni-restoremode.bb

```diff
@@ -40,7 +40,6 @@ RDEPENDS_${PN} += "\
 "
 
 RRECOMMENDS_${PN}_x64 = "\
-	kernel-module-tpm-tis \
```

Comment:
I think the `tpm_tis` module is generally useful for targets which have TPMs (like PXIs) do we really want to remove it in all cases? 

@gratian ?